lib/src/core/di/provider.dart

```diff
@@ -162,12 +164,22 @@ class Provider {
   bool get multi => _multi ?? false;
 }
 
-/// Creates a [Provider].
+/// Creates an injector [Provider] for the given [token] based on a given
 ///
-/// To construct a [Provider], bind a `token` to either a class, a value, a
-/// factory function, or to an existing [token].
+/// - Class
+/// - Value
+/// - Factory function, or
+/// - Another token
 ///
-/// The `token` is most commonly a class or [OpaqueToken-class.html].
+/// The [token] is most commonly a class or an opaque token. More details
```

Comment:
Let's avoid the "can be found" indirectness, and be clearer about where the link goes. How about:

For more details about providers, see [Injector providers][di] in the Dependency Injection page.